spec/unit/pops/parser/lexer2_spec.rb

```diff
@@ -702,4 +702,43 @@ def expect_issue(code, issue)
       expect_issue('"asd', Puppet::Pops::Issues::UNCLOSED_QUOTE)
     end
   end
+
+end
+
+describe Puppet::Pops::Parser::Lexer2 do
+
+  include PuppetSpec::Files
+
+  # First line of Rune version of Rune poem at http://www.columbia.edu/~fdc/utf8/
+  # characters chosen since they will not parse on Windows with codepage 437 or 1252
+  # Section 3.2.1.3 of Ruby spec guarantees that \u strings are encoded as UTF-8
+  # ᚠᛇᚻ᛫ᛒᛦᚦ᛫ᚠᚱᚩᚠᚢᚱ᛫ᚠᛁᚱᚪ᛫ᚷᛖᚻᚹᛦᛚᚳᚢᛗ
```

Comment:
The comments here are just a sanity check copy of the actual Rune characters specified with `\uXXXX` style code points so that if you need to see what they actually look like you can compare.  I don't think there's anything particularly wrong with including this, even if your current app isn't properly rendering them - https://en.wikipedia.org/wiki/Runic_(Unicode_block)#Fonts for which fonts carry the appropriate characters.  Usually when fonts miss characters, applications follow a font substitution protocol...

I specifically chose these rune characters as they cannot easily be converted to a local codepage on Windows. In my experience, I have no issues in editors (note that I use `Source Code Pro` as my editor font which doesn't include these characters) or nearly all browsers:

### Vim on Mac

![screen shot 2016-02-05 at 11 37 02 am](https://cloud.githubusercontent.com/assets/394081/12857108/3059defa-cbfd-11e5-8c46-a141bba5f37d.png)

### Sublime on Windows

![screen shot 2016-02-05 at 11 39 02 am](https://cloud.githubusercontent.com/assets/394081/12857112/34aaa548-cbfd-11e5-8916-2095849e0ef5.png)

### Chrome on Windows

![screen shot 2016-02-05 at 11 43 11 am](https://cloud.githubusercontent.com/assets/394081/12857230/cc003f66-cbfd-11e5-9a9d-4c81053555e4.png)

### Safari on Mac

![screen shot 2016-02-05 at 11 45 55 am](https://cloud.githubusercontent.com/assets/394081/12857270/0bf6b6e0-cbfe-11e5-8d7f-660fdfe6809e.png)

### FireFox on Mac

![screen shot 2016-02-05 at 11 51 50 am](https://cloud.githubusercontent.com/assets/394081/12857402/e616777a-cbfe-11e5-85db-8801418a38ec.png)


The only place where I'm actually seeing a problem is the Mac version of Chrome, and I don't think it's worth changing this for what appears to be a browser bug.

Ultimately however, the comment isn't strictly necessary, so if you're really hung up on it, it can be removed - but I'd prefer not to.